src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const trsOfTbody = [...tbody.querySelectorAll('tr')];
+const tfoot = document.querySelector('tfoot');
+
+thead.querySelector('tr')
+  .insertBefore(thead
+    .querySelector('tr')
+    .children[1]
+    .cloneNode(true),
+  thead
+    .querySelector('tr')
+    .children[4]);
+
+for (const tr of trsOfTbody) {
```

Comment:
if you use all trs here not just trsOfBody, you can avoid code for `thead` and `tfoot`